src/components/UpdateArticleVotes.jsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react'
+import { getSingleArticle, patchArticleVotes } from '../api'
+
+const UpdateArticleVotes = ({id}) => {
+    const [votes, setVotes] = useState(0)
+    const [isError, setIsError] = useState(false)
+
+    useEffect(() => {
+        getSingleArticle(id)
+        .then((article) => {
+            setVotes(article.votes)
+            setIsError(false)
+        })
+        .catch((err) => {
+            setIsError(true)
+        })
```

Comment:
You already have this data in your `article` state. Since you render this component as a child of `SingleArticle`, you could pass `article.votes` down on props. When `article.votes` gets updated in this component, it will all automatically rerender 😄